web/Views/Landlord/Profile.cshtml

```diff
@@ -20,7 +20,7 @@
         <div class="mb-4">
             <div class="d-flex">
                 <h3 class="font-h3 d-inline-block">Personal details</h3>
-                <a class="bi bi-pencil ms-auto" onclick="location.href='@Url.Action("EditProfilePage", "Landlord", new { landlordId = Model.LandlordId })'; setCurrentTab(0);"></a>
+                <a href="#" class="bi bi-pencil ms-auto" onclick="location.href='@Url.Action("EditProfilePage", "Landlord", new { landlordId = Model.LandlordId })'; setCurrentTab(0);"></a>

```

Comment:
was this meant to be a partn of this commit?

Reply:
I thought I'd fix this styling issue that had been bugging me while I was fixing the main issue, but it probably should've been a separate commit